build-logic/src/main/kotlin/com/plusmobileapps/chefmate/convention/KmpLibraryConventionPlugin.kt

```diff
@@ -83,7 +83,8 @@ class KmpLibraryConventionPlugin : Plugin<Project> {
 
                 // Configure JVM target
                 jvm()
-
+                compilerOptions.optIn.add("kotlin.time.ExperimentalTime")
```

Comment:
- [ ] Replace with kotlinx.datetime

Reply:
https://github.com/Kotlin/kotlinx-datetime?tab=readme-ov-file#deprecation-of-instant